### PR TITLE
Add link to legacy app for carbon offsets

### LINKED ISSUE #2035: [offset] Update RA offset page copy to direct user to old build site
The form on https://app.klimadao.finance/#/offset was deprecated to shrink maintainable code footprint and to encourage users to retire on Carbonmark.

There are still a small subset of users who want to retire using pool tokens asset that they own (for example, retire carbon using BCT). After taking down the offset form in the RA, there were a number of complaints in the #klima-chat channel on discord.

There was an internal discussion is this [discord thread](https://discord.com/channels/1135827610434289727/1184579974993281064).

The product team discussed and agreed to following changes:

- [x] Take an older build (https://klimadao-3rkq7ymia-klimadao.vercel.app/#/offset) and update the URL domain to `legacy-app.klimadao.finance` that points to that branch (@Atmosfearful)
- [x]  On the current /offset page (https://app.klimadao.finance/#/offset) add a new paragraph to the text component that says:

> For users who want to offset paying with a pool token, please use [this site](https://legacy-app.klimadao.finance/#/offset).

### PR DESCRIPTION
**Pull request by biwano**
## Description

Adds a link to legacy app for carbon offsets

## Related Ticket

Closes #2035 

## Notes For QA

Relevant preview URLs:
- https://klimadao-app-git-2035offsetpagelink-klimadao.vercel.app/#/offset

Other notes:
- I used the link to https://klimadao-3rkq7ymia-klimadao.vercel.app/#/offset instead of https://legacy-app.klimadao.finance/#/offsets because the offset page is not available on the latest.
- I hesitated to put a link that opens in another tab, LMK what you prefer.

### CODE REVIEW

**Review by jabby09 — COMMENTED** [review]
@biwano this is working but I want to wait until we resolve the issue on the new domain URL

File: app/components/views/Offset/index.tsx [hunk 1]
```diff
@@ -41,6 +41,12 @@ export const Offset = (props: Props) => {
                 live implementation.
               </Trans>
             </Text>
+            <Text t="caption" className={styles.ctaCard_header_subtitle}>
+              <Trans>
+                For users who want to offset paying with a pool token, please
```
Comment (sprrwhwk) [inline app/components/views/Offset/index.tsx:46]:
```suggestion
                Users who want to offset via pool tokens can do so using
```

**Review by sprrwhwk — APPROVED** [review]

File: app/components/views/Offset/index.tsx (hunk 1 above)
Comment (biwano) [reply app/components/views/Offset/index.tsx:46]: @jabby09

**Comment by biwano** [comment]
> @biwano this is working but I want to wait until we resolve the issue on the new domain URL

@jabby09 I fixed the issue with the new domain. Hopefully saving @Atmosfearful some precious minutes :)

**Review by jabby09 — CHANGES_REQUESTED** [review]
@biwano on https://klimadao-app-git-2035offsetpagelink-klimadao.vercel.app/#/offset

this site...is resolving to https://klimadao-3rkq7ymia-klimadao.vercel.app/#/offset not https://legacy-app.klimadao.finance/

**Comment by biwano** [comment]
> @biwano on https://klimadao-app-git-2035offsetpagelink-klimadao.vercel.app/#/offset
> 
> this site...is resolving to https://klimadao-3rkq7ymia-klimadao.vercel.app/#/offset not https://legacy-app.klimadao.finance/

Sorry about that @jabby09 . I made two commits in quick succession. I checked the last one but github got confused and promoted the first one. I fored a redeployment in vercel so now github should point to the right deployment.

**Review by jabby09 — APPROVED** [review]

**Pull request merged**